### ISSUE TITLE
Issues with json_prepared() method

### ISSUE THREAD

**Issue by dimir** [whosgonna/Zabbix-Tiny.pm#27]
I have found 2 issues with that. First, it's misspelled in example:
```
$zabbix->prepare('host.get', $params);  # Prepare the query.
print $zabbix->prepared . "\n";         # Get the JSON query without actually executing it.
my $host = $zabbix->do;                 # Execute the prepared query.
```
Secondly, I couldn't find mention of the method in the manual in section DEBUGGING METHODS.

**Comment by whosgonna** [comment]
I'll work on updating the documentation, but the example should be this:

```perl
print $zabbix->json_prepared . "\n";         # Get the JSON query without actually executing it.
```

```prepare```  is a method that builds the json body to be sent. 
```json_prepared``` is an attribute that stores the json.

A few thoughts:

- Note that if you run ```do``` with an argument, it will run ```prepare``` transparently.   If you run ```do``` with no argument, then the most recently prepared method will be used.
- Because ```do``` handles this automatically, there's not much real-world need to use either ```prepare``` or ```json_prepared```, etc. 
- In retrospect, it would have been smarter to have used an attribute of ```message``` with ```set_message``` and ```get_message``` as the setter and getter respectively.  If I have time I might look into updating the module to use those methods (but still retaining the ```prepare``` and ```json_prepared methods for backwards compatibility).


Next Step:  Update Synopsis section of POD.

**Comment by whosgonna** [comment]
Fixed in  3da3aad

**Comment by dimir** [comment]
Looks like there's one copy-paste issue, see my inline comment in:

https://github.com/whosgonna/Zabbix-Tiny.pm/commit/3da3aad42abf9fee33cb0bf66ad9df51f89485c8#r28235155
